### PR TITLE
NEWS.md: interim update

### PR DESCRIPTION
**Pull request by johanmalm**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -30,6 +30,80 @@ The format is based on [Keep a Changelog]
...
+## [unreleased]
+
+The main focus in this release has been to port labwc to wlroots 0.18 and to
+grind out associated regressions. Nonetheless, it contains a few non-related
+additions and fixes as described below.
+
+As a one-off note, there appears to be an issue with increased commit failures
+when using wlroots-0.18, particularly with intel drivers. If this turns out to
+be an issue for anyone please try running with `WLR_DRM_NO_ATOMIC=1` or revert
+to the v0.17 branch until this is resolved.
```
Comment (Consolatis) [inline NEWS.md:42]: `or run the labwc v0.7 branch or its latest release until this is resolved` ?

File: NEWS.md [hunk 2]
```diff
@@ -30,6 +30,80 @@ The format is based on [Keep a Changelog]
...
+
+A v0.7 branch has been created for bug fixes beyond `0.7.3` (built with wlroots
+`0.17`.
```
Comment (Consolatis) [inline NEWS.md:45]: `beyond 0.7.4`

File: NEWS.md [hunk 3]
```diff
@@ -30,6 +30,80 @@ The format is based on [Keep a Changelog]
...
+
+A big thank you goes to @Consolatis for carefully crafting the commits to port
+across to wlroots 0.18.0. Many thanks also to the other core devs @ahesford,
+@jlindgren90 and @tokyo4j for reviewing, merging as well as contributing many
```
Comment (Consolatis) [inline NEWS.md:49]: `s/@jlindgren90/@jlindgren90, @johanmalm/`

File: NEWS.md (hunk 1 above)
Comment (johanmalm) [reply NEWS.md]: Done

File: NEWS.md (hunk 2 above)
Comment (johanmalm) [reply NEWS.md]: Done

File: NEWS.md (hunk 3 above)
Comment (johanmalm) [reply NEWS.md]: :P

**Pull request merged**